examples/with-typescript/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+.next
```

Comment:
I think it's worth adding package.lock and yarn.lock as well

Reply:
Lockfiles should always be committed to source control!